spec/acceptance/bundle_management_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper_acceptance'
+require 'fileutils'
+
+describe 'Managing Gemfile dependencies' do
+  before(:all) do
+    # Do everything in a clean tmpdir.
+    @old_pwd = Dir.pwd
+    @tmpdir = Dir.mktmpdir('pdk-acceptance')
+
+    Dir.chdir(@tmpdir)
+
+    # We need a module to test inside of.
+    shell_ex("#{path_to_pdk} new module test_module --skip-interview")
+  end
+
+  after(:all) do
+    Dir.chdir(@old_pwd)
+    FileUtils.remove_entry_secure(@tmpdir, true)
+  end
+
+  context 'when there is no Gemfile.lock' do
+    before(:all) do
+      shell_ex('rm -f test_module/Gemfile.lock')
+    end
+
+    after(:all) do
+      shell_ex('rm -f test_module/Gemfile.lock')
+    end
+
+    # TODO: come up with a way to invoke only the bundler stuff without trying to run unit tests
+    describe command("cd test_module && #{path_to_pdk} test unit") do
```

Comment:
I've checked serverspec's source, and there doesn't seem to be any way to specify any runtime environment (like CWD) to the `command()`.